<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OPOLaserControlControls</class>
 <widget class="QWidget" name="OPOLaserControlControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>524</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Port</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>201</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QComboBox" name="comboBoxPortType">
             <item>
              <property name="text">
               <string>COM</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyUSB</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>/dev/ttyS</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxPort">
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="value">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="buttonConnect">
           <property name="toolTip">
            <string>Do image processing</string>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonStatus">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Get Status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelState">
           <property name="text">
            <string>STATE</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Custom Message</string>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>201</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="lineMessage"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSendCustomMessage">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="browserResponses"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
